membership.py

```diff
@@ -0,0 +1,207 @@
+    parser.add_option(
+        '--copy-qtgui',
+        action='store_true',
+        dest='copy',
+        default=False)
+    parser.add_option(
+        '--generate-common-members',
+        action='store_true',
+        dest='generate',
+        default=False)
+    parser.add_option(
+        '--sort-common-members',
+        action='store_true',
+        dest='sort',
+        default=False)
+    (options, args) = parser.parse_args()
+
+    if options.copy:
+        copy_qtgui_to_qtwidgets()
+
+    elif options.generate:
+        generate_common_members()
+
+    elif options.sort:
+        sort_common_members()
+
+    else:
+
+        # Import <binding>
+        binding = __import__(options.binding)
+
+        for importer, modname, ispkg in pkgutil.walk_packages(
+                path=binding.__path__,
+                prefix=binding.__name__ + '.',
+                onerror=lambda x: None):
+            if modname not in SKIP_MODULES:
+                MODULES.append(modname)
+                basemodule = modname[:modname.rfind('.')]
+                submodule = modname[modname.rfind('.')+1:]
+                try:
+                    import_statement = (
+                        'from ' + basemodule + ' import ' + submodule)
+                    exec(import_statement)
+                    # print(import_statement)
+                except (ImportError, AttributeError, SyntaxError) as error:
+                    # SyntaxError catched here because e.g. _port3
+                    # running on Python 2...
+                    print('WARNING: Skipped import', modname, error)
+
+                try:
+                    raw_members = []  # avoid Hound errors
+                    exec('raw_members = dir(' + submodule + ')')
+                    members = []
+                    for member in raw_members:
+                        if member not in SKIP_MEMBERS and \
+                                not member.startswith('_'):
+                            try:
+                                import_statement = (
+                                    'from ' + basemodule + '.' + submodule +
+                                    ' import ' + member)
+                                exec(import_statement)
+                                # print(import_statement)
+                                MODULES.append(modname + '.' + member)
+                            except (AttributeError, SyntaxError) as error:
+                                # SyntaxError catched here because e.g. _port3
+                                # running on Python 2...
+                                print('WARNING: Skipped import',
+                                      modname, error)
+                except (NameError) as error:
+                    print('WARNING: Skipped dir() command', modname, error)
+
+        # Remove duplicates and sort
+        MODULES = sorted(list(set(MODULES)))
+
+        # Print all modules (for debugging)
+        # for module in MODULES:
```

Comment:
How about putting this and the `pprint` below under an `if` for [`QT_VERBOSE`](https://github.com/mottosso/Qt.py/blob/07f631545570c62310248e2a2545ea3103e811bc/Qt.py#L52)?

Reply:
That's a nice idea. I'll make that change here now.